File: Java/nilai.java

import java.util.Scanner;
public class nilai {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.print("Masukkan nilai tugas: ");
        double tugas = sc.nextDouble();
        System.out.print("Masukkan nilai uts: ");
        double uts = sc.nextDouble();
        System.out.print("Masukkan nilai uas: ");
        double uas = sc.nextDouble();
        double nilaiAkhir = (0.3 * tugas) + (0.3 * uts) + (0.4 * uas);

        System.out.println("Nilai Akhir: " + nilaiAkhir);

        if (nilaiAkhir >= 60) 
            System.out.println("LULUS");
        else    
            System.out.println("TIDAK LULUS");

    }
}

File: cpp/nilai.cpp

#include <iostream>
using namespace std;

// pernghitngan nilai menggunakan bahasa c++
int main() {
    float tugas, uts, uas, nilaiAkhir;
    cout << "Masukkan nilai Tugas: ";
    cin >> tugas;
    cout << "Masukkan nilai UTS: ";
    cin >> uts; 
    cout << "Masukkan nilai UAS: ";
    cin >> uas;

    nilaiAkhir = (0.3 * tugas) + (0.35 * uts) + (0.35 * uas);
    cout << "Nilai Akhir: " << nilaiAkhir << endl;
    if(nilaiAkhir >= 60)
        cout << "Keterangan: Lulus" << endl;
    else
        cout << "Keterangan: Tidak Lulus" << endl;
}
